### PR TITLE
fix(DB/Creature): Fire Nova Totem VII - using wrong Fire Nova spell

### PR DESCRIPTION
**Pull request by avarishd**
I may or may not have been slapped hard by that on CC.

Additionally, we are safe to assume that other Fire Nova Totems (1-9) use the corresponding spell rank.

## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/16097
- Closes https://github.com/chromiecraft/chromiecraft/issues/5510

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [x] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

Thanks Pepsiman @heyitsbench 

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

target any hostile mob
.cast back 32062 t
check damage

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
Isn't the `creature_template_spell` table used exclusively for vehicles/mind control situations?

File: data/sql/updates/pending_db_world/rev_1698326815078104700.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-- Fire Nova Totem VII - Cast Fire Nova (Rank 7)
+DELETE FROM `creature_template_spell` WHERE (`CreatureID` = 15483);
+INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES
+(15483, 0, 25537, 42328);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1698326815078104700.sql:4]: don't be lazy, use UPDATE instead

Comment (avarishd) [reply data/sql/updates/pending_db_world/rev_1698326815078104700.sql:4]: Using update is lazier.

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1698326815078104700.sql:4]: the entry already exist, update is the correct way to tackle this.

Comment (avarishd) [reply data/sql/updates/pending_db_world/rev_1698326815078104700.sql:4]: sigh

**Review by Nyeriah — APPROVED** [review]

**Pull request merged**